Data/ByteString/Lazy/Internal/Deque.hs

```diff
@@ -1,5 +1,5 @@
 {- |
- A Deque used for accumulating `S.ByteString`s in `Data.ByteString.Lazy.dropEnd`.
+ A Deque used for accumulating `S.StrictByteString`s in `Data.ByteString.Lazy.dropEnd`.
```

Comment:
I think I've been nerd-sniped into killing off this module entirely. But I'll wait until after this patch lands.